scicamera/controls.py

```diff
@@ -72,6 +54,14 @@ def set_controls(self, controls: dict | Controls):
         else:
             raise RuntimeError(f"Cannot update controls with {type(controls)} type")
 
+    @property
+    def FrameRate(self) -> float:
+        return _durations_to_framerates(self.FrameDurationLimits)
+
+    @FrameRate.setter
+    def FrameRate(self, value: float) -> None:
+        self.FrameDurationLimits = _framerates_to_durations(value)
```

Comment:
Can we change the name here? Should probably also ensure that it's always calls using a`Tuple[float, float]` to avoid any chance of type polymorphism.

```suggestion
        self.frame_duration_limits = _framerates_to_durations((value, value))
```